Bottom: 1bf506765f53efb01fdd8c2fcdc76a5f63ee3cab
Top:    b8ad14dd30ae111af765674400abdd4be68e4de7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-19 11:33:45 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
